R/chunkedUploadFile.R

```diff
@@ -116,6 +116,7 @@ createChunkedFileUploadToken<-function(filepath, contentType) {
   chunkedFileTokenRequest<-list(
     fileName=basename(filepath),
     contentType=contentType,
+    s3UploadDestination=createListFromS4Object(s3UploadDestination),
```

Comment:
consider naming this uploadDestination in case we support other providers (ex Google Cloud Storage)?